apps/core/components/ItineraryCard/ItineraryCard.js

```diff
@@ -107,5 +122,35 @@ const styles = StyleSheet.create({
     borderColor: defaultTokens.borderColorCard,
     borderBottomWidth: 1,
     borderTopWidth: 1,
+    web: {
+      alignSelf: 'center',
+      flexDirection: 'row',
+      alignItems: 'center',
+      width: '100%',
+      maxWidth: 720,
+      padding: parseInt(defaultTokens.spaceMedium, 10),
+      marginBottom: parseInt(defaultTokens.spaceMedium, 10),
+      borderBottomWidth: 0,
+      borderTopWidth: 0,
+      borderRadius: parseInt(defaultTokens.borderRadiusNormal, 10),
+      boxShadow: '0 2px 4px 0 rgba(23,27,30,.1)',
+      transitionDuration: '0.3s',
+      transitionProperty: 'box-shadow',
+      transitionTimingFunction: 'ease-out',
+    },
+  },
+  sectors: {
+    web: {
+      flex: 1,
+    },
+  },
+  price: {
+    web: {
+      color: '#2e353b',
```

Comment:
This color is not in orbit, but maybe it should be there. What do you think? If you agree we could leave here some TODO comment

Reply:
Ok, I just replaced this component with `LocalizedPrice` component so I also removed this style. But I'll be updating `LocalizedPrice` with this style so I'll include this comment there.